apps/betterangels-backend/shelters/schema.py

```diff
@@ -0,0 +1,30 @@
+from typing import cast
+
+import strawberry
+import strawberry_django
+from django.db.models import Prefetch
+from shelters.models import ExteriorPhoto, InteriorPhoto, Shelter
+from shelters.types import ShelterType
+from strawberry_django.pagination import OffsetPaginated
+
+
+@strawberry.type
+class Query:
+    shelter: ShelterType = strawberry_django.field()
+
+    @strawberry_django.offset_paginated()
+    def shelters(self) -> OffsetPaginated[ShelterType]:
+        shelters = Shelter.objects.prefetch_related(
+            Prefetch(
+                "exterior_photos",
+                queryset=ExteriorPhoto.objects.order_by("created_at"),
+                to_attr="prefetched_exterior_photos",
```

Comment:
will this cause every query to prefetch?

Reply:
you mean even if heroImage isn't requested? yes. it only bumps the query count by 2, which is considerably cheaper than most of our queries